sdk/search/azure-search-documents/azure/search/documents/_internal/_search_client.py

```diff
@@ -59,6 +60,8 @@ class SearchClient(HeadersMixin):
     :type index_name: str
     :param credential: A credential to authorize search client requests
     :type credential: ~azure.core.credentials.AzureKeyCredential
+    :keyword str api_version:
+        The Search API version to use for requests. Default value is "2019-05-06-Preview".
```

Comment:
Is it a good idea to document the default value? It's going to keep changing and eventually we "may" forget to update it.

Reply:
That's the point. :)

We want user to get this information in intellisence (if they have) and know what is current API version they are using.

Reply:
I think the point of @rakshith91 is that the doc is then hard-coded with the API version, which I think is a fair point.

Reply:
What I thought was it would have different values for different version of Search SDK. 

But I am also ok to remove it from the docstring.